__tests__/patchArticleVotes.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("should return an object of corresponding article_id with vote property updated", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({
+        inc_votes: 32,
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+      });
+  });
+
+  it("should return a 400 error if missing inc_votes property in body", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Missing required field: votes");
```

Comment:
can we alter this test to be sending a body with a key of something other than inc_votes? what happens if we send {incvotes: 1} for example? We want to make sure that it still gets rejected